ykrt/src/mt.rs

```diff
@@ -134,6 +135,9 @@ impl MT {
                 .map_err(|e| format!("Invalid sidetrace threshold '{s}': {e}"))?,
             Err(_) => DEFAULT_SIDETRACE_THRESHOLD,
         };
+        // Initialise a jitdump file, if required.
+        #[cfg(target_os = "linux")]
+        ensure_jitdump_initialised();
```

Comment:
Can't the static initialiser in the jitdump module do this, rather than forcing mt to know about it?

Reply:
But that's nonsense, isn't it, because we are going to have to check if it's initialised to write a record! Yep, OK!